### ISSUE TITLE
Odoo V9

### ISSUE THREAD

**Issue by mercadolino** [alien-group/transport_sale#2]
Hello, one year ago you mentioned you would port this module to Odoo V9. Any news about that?
I saw your youtube video about this module and it is great for a transport company you did a terrific job! Can't wait to see it for Odoo 9

**Comment by aliencrash** [comment]
We have migrated to version 10. We are finalizing it and testing.

**Comment by aliencrash** [comment]
[apresentacaofleetV1.3_english.pdf](https://github.com/alien-group/transport_sale/files/603963/apresentacaofleetV1.3_english.pdf)
Here you can see a short presentation.

**Comment by mercadolino** [comment]
Awesome! Ready to download?

**Comment by aliencrash** [comment]
No. Not yet. We will sell this modules because. we are adding more features.

**Comment by macao100** [comment]
Hello
Please how can i by this module at what price and that is ready to work in french.
Thank you

**Comment by aliencrash** [comment]
Hi,

I will make a new video, we only have for version 10. The module is
developed in English, it can be translated. In attachment is part of the
new version.

Regards

ALIEN GROUP LDA <http://www.alien-group.com> 	

Alaney Dória

CEO | Odoo Consultant

em.alaney.doria@alien-group.com

tm.+244 913 728 600

Linkedin <https://www.linkedin.com/in/alaneyd> Facebook
<http://www.facebook.com/aliengroup> Facebook
<http://www.twitter.com/ohomemeamente>
	Código QR

Rua Doutor Agostinho Neto, 156, Bairro Azul, Luanda - Angola, Tel: +244
913 728 600 | +244 913 728 650,
Website: www.alien-group.com <http://www.alien-group.com>, Email:
geral@alien-group.com <mailto:geral@alien-group.com>

Não imprima este email caso não seja estritamente necessário. A Terra
agradece-lhe!

**Comment by morrismusumi** [comment]
Hello,

What is the progress on this module and how can i get it.

thanks
